fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -139,6 +145,36 @@ export const LocationSuggest = forwardRef<HTMLInputElement, Props>(
       getLocationSuggest,
     ]);
 
+    const loadInitialLocation = useCallback(async () => {
+      if (!initialValue || initialLocation?.id.value === initialValue) {
+        return;
+      }
+
+      if (!client) {
+        return;
+      }
```

Comment:
D'oh, I didn't realise this was optional! I think we should make `client` a required prop now, otherwise it'll be confusing when we hit this code path and the feature mysteriously doesn't work.

Reply:
Thank you for re-writing the story part to make this work. Do we also need to remove the default usage scenario from here: https://github.com/seek-oss/wingman/tree/master/fe/lib/components/LocationSuggest#default-usage

Maybe make Usage with Apollo Client the default scenario?

Reply:
I think we can update the default usage example with `useApolloClient`, which is doing the same thing as is described in the comment:

> // Higher component in tree wraps children in apollo provider